src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/modifier/interfacememberimpliedmodifier/InputInterfaceMemberImpliedModifierSealedClasses.java

```diff
@@ -0,0 +1,55 @@
+/*
+InterfaceMemberImpliedModifier
+
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.modifier.interfacememberimpliedmodifier;
+
+public interface InputInterfaceMemberImpliedModifierSealedClasses {
+    String s = null;
+    // 3 violations above:
+    //                  'Implied modifier 'final' should be explicit.'
+    //                  'Implied modifier 'public' should be explicit.'
+    //                  'Implied modifier 'static' should be explicit.'
+    void m();
+    // 2 violations above:
+    //                  'Implied modifier 'abstract' should be explicit.'
+    //                  'Implied modifier 'public' should be explicit.'
+    sealed interface J permits A {
+    // 2 violations above:
+    //                  'Implied modifier 'public' should be explicit.'
+    //                  'Implied modifier 'static' should be explicit.'
+        void m();
+        // 2 violations above:
+        //                  'Implied modifier 'abstract' should be explicit.'
+        //                  'Implied modifier 'public' should be explicit.'
+    }
+    sealed class B permits A, D { }
+    // 2 violations above:
+    //                  'Implied modifier 'public' should be explicit.'
+    //                  'Implied modifier 'static' should be explicit.'
+    non-sealed class D extends B { }
+    // 2 violations above:
+    //                  'Implied modifier 'public' should be explicit.'
+    //                  'Implied modifier 'static' should be explicit.'
+}
+
+interface InputInterfaceMemberImpliedModifierSealedClassesCorrected {
+    final public static String s = null;
+    abstract public void m();
+    public  static  sealed interface J permits A {
+        abstract public void m();
+    }
+    public static sealed class B permits D { }
+   public static non-sealed class D extends B { }
```

Comment:
Indentation is off.

Reply:
done

Reply:
@mahfouz72 Not done. Indentation is worse now.

This seems to be a constant issue in PRs. You may want to keep a closer eye on this.

Reply:
I hope it is done now :)